content/ux-strat-bootcamp/week-one/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Week One
+---
+
+# Welcome to Bootcamp
+
+#### Day 1
+- **Interviews + User Journey Mapping**
+
+{{% contentlink path="ux-strat-bootcamp/week-one/monday" %}}
```

Comment:
Looking at the upcoming bootcamp, it does actually start on a tuesday.  So this is probably going to confuse people. If it has a 10% chance of confusing people, it'sll confuse 10% of the people.

How about this:
Dont record day1 day 2, monday, tuesday, or anything like that ni the sllabus itself. Just manage that stuff in standup? It'll make this file structure way simpler and more versatile.